src/app/article/article.component.ts

```diff
@@ -64,6 +68,10 @@ export class ArticleComponent implements OnInit {
     }
   }
 
+  trackByFn(index, item) {
```

Comment:
Whats the rational behind using index for tracking all lists?

Reply:
I wasn't sure which one of them actually had an id, so I used index as the easiest way to track them.